discover/deployment/articles/02-configuring-liferay/02-document-repository.markdown

```diff
@@ -222,10 +222,11 @@ You must include this property in `system-ext.properties`:
     org.xml.sax.driver=com.sun.org.apache.xerces.internal.parsers.SAXParser
 
 Other app servers also need this configuration if they don't contain a
-`SAXParser`. 
+`SAXParser`. Remember to place the `system-ext.properties` in your @product@
+installation's `/WEB-INF/classes` directory.
 
```

Comment:
@chrislui  Some app servers (like WebLogic) don't have the concept of the exploded WAR that's available on the file system. In that case, you'd have to open the WAR file, modify system.settings, re-zip it, and redeploy it. It's incredibly inconvenient and highly kludgey, and we try to avoid instructions that tell people to do that if there's any other way.